source/docs/api-reference/device-specific/talonfx/open-loop-requests.rst

```diff
@@ -5,13 +5,19 @@ Open-Loop control typically refers to directly controlling device output.
 
 There are open-loop control requests for all TalonFX control output types. With the exception of FOC-only control requests, all open-loop control requests follow the naming pattern ``{ControlOutputType}Out``. For example, the open-loop Voltage control request is called ``VoltageOut``. FOC-only control requests follow the naming pattern ``{ControlOutputType}``.
 
+In the below example, note that devices are initialized with two arguments. These arguments correspond to :ref:`the device ID <docs/tuner/device-details-page:configuring name & ids>` and CAN bus name. For CANivore, this is the name of the CANivore as configured in :doc:`/docs/tuner/index`. For more information, see the :doc:`CANivore API </docs/canivore/canivore-api>` documentation.
```

Comment:
I'm not sure if this is the right place to put this. Maybe put this in API Overview? Or we actually create a page documenting creating devices. We need somewhere to document thread safety anyways.

Reply:
This can be changed in a separate PR.